src/FluentMigrator.Runner.MySql/Processors/MySql/MySqlDbFactory.cs

```diff
@@ -24,6 +24,7 @@ public class MySqlDbFactory : ReflectionBasedDbFactory
         {
             new TestEntry("MySql.Data", "MySql.Data.MySqlClient.MySqlClientFactory"),
             new TestEntry("MySqlConnector", "MySql.Data.MySqlClient.MySqlClientFactory"),
+            new TestEntry("MySqlConnector", "MySqlConnector.MySqlConnectorFactory")
```

Comment:
Please add a comma at the end of the line.

Reply:
added!